### PR TITLE
Fix JWK format support of OIDC public key

### PR DESCRIPTION
**Pull request by faraway**
@m00s @patfrat  Please review
I was mostly using `PEM` certs and overlook this issue. @patfrat found out this problem in his environment, and proposed a solution. The detailed discussion is here: https://github.com/angularjs-oauth/oauth-ng/issues/112. Test is also added.

### CODE REVIEW

File: dist/oauth-ng.js [hunk 1]
```diff
@@ -443,7 +443,7 @@ accessTokenService.factory('AccessToken', ['Storage', '$rootScope', '$location',
       return;
     }
     var time = (new Date(service.token.expires_at))-(new Date());
-    if(time && time > 0){
+    if(time && time > 0 && time <= 2147483647){
```
Comment (faraway) [inline dist/oauth-ng.js]: hmm...it seems the built script and source is not in sync.

**Comment by patfrat** [comment]
Looks good for me ;)

**Comment by m00s** [comment]
@faraway @patfrat Thanks guys! Gonna merge

**Comment by patfrat** [comment]
Thanks too !! Pulled and working with my project !

**Pull request merged**